src/views/ForgotPasswordPageChange/index.js

```diff
@@ -1,31 +1,63 @@
+import AWS from 'aws-sdk';
 import React, { Component } from "react";
 import Form from "react-bootstrap/Form";
 import Button from "react-bootstrap/Button";
 import { Link, Route } from "react-router-dom";
 import LoginPage from "../LoginPage";
 
+const awsRegion = process.env.REACT_APP_AWS_REGION
 
+AWS.config.update({region:awsRegion});
+const cognitoidentityserviceprovider = new AWS.CognitoIdentityServiceProvider();
```

Comment:
just for increased readability, write this in snake case so `const cognitoIdentityServiceProvider = ...`

Reply:
Fixed